lib/LANraragi/Plugin/Metadata/Fakku.pm

```diff
@@ -24,7 +24,7 @@ sub plugin_info {
         type        => "metadata",
         namespace   => "fakkumetadata",
         author      => "Difegue",
-        version     => "0.5.1",
+        version     => "0.6",
         description => "Searches FAKKU for tags matching your archive.",
```

Comment:
Since we now have cookie support, it might be worth it to edit the description to say you'll only get controversial content metadata if you login.  

 ```suggestion
        description => "Searches FAKKU for tags matching your archive. If you have an account, don't forget to enter the matching cookie in the login plugin to be able to access controversial content.",
```